arclightning_backend/toy_client.py

```diff
@@ -0,0 +1,11 @@
+import urllib.request
```

Comment:
If you want to commit this, put it in a test_scripts folder. Also, why both urllib and requests??